pgrx-pg-config/src/lib.rs

```diff
@@ -67,6 +67,7 @@ pub enum PgMinorVersion {
     Release(u16),
     Beta(u16),
     Rc(u16),
+    Devel,
```

Comment:
We don't need this and I don't want it.  :)

When pg18beta1 is released we can use the `::Beta(1)` variant, which is how we've handled this in the past.